website_snippet_anchor/static/src/js/anchor_option.js

```diff
@@ -0,0 +1,80 @@
+/* Copyright 2015, 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). */
+
+"use strict";
+odoo.define("website_snippet_anchor.anchor_option", function (require) {
+    var core = require("web.core"),
+        options = require("web_editor.snippets.options"),
+        utils = require("website.utils"),
+        sprintf = _.str.sprintf,
+        _t = core._t;
+
+    // Option to have anchors in snippets
+    options.registry.anchor = options.Class.extend({
+
+        anchor_ask: function(type) {
+            if(type !== false) {
```

Comment:
type is the event type. You want to react only in `click` event, so leave it as it was: `if (type !== "event")`

Reply:
No, when on click type is in 11.0 false. I'll remove it when I apply the attribute data-no-preview